gamemodes/horde/entities/entities/horde_watchtower/init.lua

```diff
@@ -68,7 +67,9 @@ function ENT:Think()
 end
 
 hook.Add("PlayerUse", "PickUpWatchtower", function(ply, ent)
-	if HORDE:IsWatchTower(ent) and ent:GetNWEntity("HordeOwner"):IsValid() and ent:GetNWEntity("HordeOwner") == ply then
+	if not HORDE:IsWatchTower(ent) then return false end
+    if not ent:GetNWEntity("HordeOwner"):IsValid() then return false end
```

Comment:
you're using tab and space indentation, please use one type of indentation in the entire file

in this case use tabs as horde uses them

Reply:
Done